requirements/optional.txt

```diff
@@ -0,0 +1 @@
+ninja
```

Comment:
If pytorch also installs ninja, shall we make it a default requirements to speed up compilation?

Reply:
Most users will use pre-compiled packages, so this library is not required. Also, if you use `conda` to install PyTorch, then ninja is automatically installed.